### PR TITLE
Change display record request history and autocomplete bugfix.

### PR DESCRIPTION
**Pull request by szok**
- Change display old/new to old -> new in Record Request History
- Autocomplete bugfix: after removal of the element does not delete text in input

### CODE REVIEW

File: ui/static/app/record-request/record-request-detail.component.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export class RecordRequestDetailComponent implements OnInit {
 
     if (recordValue !== recordRequestValue) {
       return `
-        <span class="old">${recordValue}</span> /
+        <span class="old">${recordValue}</span> ->
```
Comment (ghost) [inline ui/static/app/record-request/record-request-detail.component.ts:44]: what's this `->`?

**Comment by ghost** [comment]
:ok:

File: ui/static/app/record-request/record-request-detail.component.ts (hunk 1 above)
Comment (szok) [inline ui/static/app/record-request/record-request-detail.component.ts:44]: Kris suggestion to change, it is more readable.

**Comment by krzysztofzygmunt** [comment]
:ok:

**Pull request merged**